addons/wasmedge/disable

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+PATCH_DIR=`dirname "$0"`/patch
+
+echo "Disabling crun with wasmedge support"
+sudo rm $SNAP_DATA/var/lock/wasmedge.enabled
```

Comment:
Instead of `sudo` please use the `run_with_sudo` function from `utils` [1].

[1] https://github.com/canonical/microk8s-addons/blob/main/addons/openebs/enable#L5